app/helpers/react_components/student/solution_summary.rb

```diff
@@ -0,0 +1,25 @@
+module ReactComponents
+  module Student
+    class SolutionSummary < ReactComponent
+      initialize_with :iteration
+
+      def to_s
+        super("student-solution-summary", {
+          iteration: SerializeIteration.(iteration),
+          is_practice_exercise: iteration.exercise.practice_exercise?,
```

Comment:
Could we change this to be `is_concept_exercise` and invert the checks in React. That will be more consistent with other places in the code base where concept exercises are the more "primary" thing.